docs/api_reference/index.rst

```diff
@@ -12,3 +12,4 @@ API Docs
    fairlearn.preprocessing
    fairlearn.reductions
    fairlearn.experimental
+   fairlearn.linear_model
```

Comment:
why is this not rendered in the docs? I don't see it.

Reply:
It is rendered for me:
![api docs index](https://user-images.githubusercontent.com/33757199/171590969-37540552-b85b-4240-ae50-030e3d9c243e.png)

Reply:
I mean on the online built one in the CI:

https://output.circle-artifacts.com/output/job/133528ab-2413-45a8-8efb-78e7084f657c/artifacts/0/docs/_build/html/main/api_reference/index.html

Reply:
FYI, merging the latest `main` should fix the issue of building the docs so that this PRs changes are visible on circleci's artifacts.